### PR TITLE
Automated cherry pick of #57238: Build and push 3.1.11 etcd image

### PR DESCRIPTION
**Pull request by xiangpengzhao**
Cherry pick of #57238 on release-1.9.

#57238: Build and push 3.1.11 etcd image

```release-note
Build etcd image for 3.1.11 version
```

### CODE REVIEW

**Review by luxas — APPROVED** [review]
/lgtm

This is not strictly necessary; to backport the release script that only Googlers can run to the older branch, but this is fine for correctness.

What's critical though is that we create a new PR to bump the 3.1.10 references in the code to 3.1.11 at HEAD for v1.9 and backport to the v1.9 branch. That is a new, standalone PR though.

**Comment by luxas** [comment]
cc @mbohlool for cherrypick approval

**Comment by mbohlool** [comment]
Please add release note

**Comment by xiangpengzhao** [comment]
@mbohlool I update the release note with the original PR title. @wojtek-t does it make sense as a release note? Any suggestions? Thanks!

**Comment by luxas** [comment]
@xiangpengzhao that's ok for me

**Comment by wojtek-t** [comment]
As @luxas already wrote, this was purely building the image and it was built once and there is no need to backport this PR to 1.9. So personally I would skip this cherrypick.
We should create a PR to bump the images to 3.1.11

**Comment by xiangpengzhao** [comment]
>So personally I would skip this cherrypick.

Agreed.

>We should create a PR to bump the images to 3.1.11

The only code I can find is here. But I had a PR #57090 to not hard-code the etcd version. Am I missing something?

https://github.com/kubernetes/kubernetes/blob/bba84d785eadbe73ff9670f22f165c0ad363b94d/test/e2e/framework/nodes_util.go#L56-L65

**Comment by wojtek-t** [comment]
That is just for upgrade tests. There are a bunch of things that needs to be bumped, in particular:
- etcd client Godeps
- pretty much all references to "3.1.10" in the codebase; the PR will be similar to this one: https://github.com/kubernetes/kubernetes/pull/49393

**Comment by mbohlool** [comment]
> So personally I would skip this cherrypick.

Closing this cherry pick to keep a clean list of cherrypicks for 1.9.1. Please reopen and get it approved today if you want it in 1.9.1. Thanks.

**Comment by xiangpengzhao** [comment]
@wojtek-t we're going to bump to 3.2.11 on HEAD (https://github.com/kubernetes/kubernetes/pull/57480), so do we need to bump to 3.1.11 before that PR get merged ? cc @jpbetz

**Comment by wojtek-t** [comment]
@xiangpengzhao - why? we can just make a PR on 1.9 branch.

**Comment by xiangpengzhao** [comment]
@wojtek-t I didn't notice we can make PR like this :) I was thinking of  that it should have a parent PR for a PR being patch to a previous release branch.

**Comment by wojtek-t** [comment]
Usually that's the procedure. But in cases where something is fixed in master in a different way, we can do a PR directly to the branch.

**Pull request closed without merging**
